zeroconf.py

```diff
@@ -41,7 +41,7 @@
 
 __author__ = 'Paul Scott-Murphy, William McBrine'
 __maintainer__ = 'Jakub Stasiak <jakub@stasiak.at>'
-__version__ = '0.17.7.dev.synthego'
```

Comment:
So https://github.com/gbiddison/python-zeroconf is a public fork related to Synthego?  Wow!

Do we have any idea if we can just use current release of [`zeroconf`](https://pypi.org/project/zeroconf/)?  Or we don't care to test if it works, and just want a quick fix with old fork.

If we want quick fix with old fork, I would pose `dev1` instead of down-numbering back to `.dev`.

Reply:
These are the cherry picked commits `@gbiddison` added
https://github.com/python-zeroconf/python-zeroconf/compare/master...gbiddison:python-zeroconf:master


Checking to see if latest `https://github.com/python-zeroconf/python-zeroconf` contains these changes or not...

Reply:
One cherry pick seems to be resolved below.
```
        # windows compatibility --> https://github.com/jstasiak/python-zeroconf/issues/84
        # testing on my osx, commenting this out has no effect on the returned list of addresses
        # if addr.get('netmask') != HOST_ONLY_NETWORK_MASK
```
https://github.com/python-zeroconf/python-zeroconf/issues/84


This cherry pick might not be needed anymore as we're using python >3.7.x
```
     # patch manually cherry-picked from stephenrauch/python-zeroconf 16db9d10dfe4cff97b609464ab24c4e71bd1aa68
    def notify_all(self):
        """Notifies all waiting threads"""
        with self.condition:
            try:
                self.condition.notify_all()
            except RuntimeError:
                # There is a bug in python 3.4 prior to 3.4.2 in which it is
                # possible for threading.notify() to raise a runtime error
                # http://bugs.python.org/issue22185

                # retry notify_all() to hopefully insure everyone gets notified
```

Reply:
Can we try pinning `zeroconf>=X` in Synthego-Brain, and seeing if tests pass?

Reply:
there are no tests we can rely on in that capacity.

Reply:
Okay gotchu.

Then if someone felt like deploying onto a beagle and HIL testing, it could be a nice tech debt removal.  Otherwise, yeah sure we can do this quick fix, in which case I think we should make it `dev1`

Reply:
I want to highlight that this package is only used in `synthego-common`
https://github.com/Synthego/Synthego-Brain/pull/1509#issuecomment-1402917965

`./synthego/common/discovery/discovery.py`
This module is used here in `Synthego-Brain`

```
grep -r discovery . --exclude-dir common                                                                                                                
./synthego/webui/web.py:from synthego.common.discovery import ServiceRegistry
./synthego/webui/modulebridge/modulebridge.py:from synthego.common.discovery import DiscoveryBrowser
./debug.py:@patch("synthego.common.discovery.discovery.socket.gethostbyname")
```

Even if we have no Python Tests, do you think we can test those functionalities manually?